kolibri/plugins/video_mp4_render/assets/src/views/index.vue

```diff
@@ -26,6 +26,8 @@
   const LangLookup = require('./languagelookup');
   const customButtons = require('./videojs-replay-forward-btns');
   const throttle = require('lodash/throttle');
+  const Lockr = require('lockr');
```

Comment:
Hrm, guess caniuse is just not showing stuff before Android 4.4! No worries then!